### ISSUE TITLE
keyboard nav not hosted on app engine

### ISSUE THREAD

**Issue by maribethb** [google/blockly#9382]
### Check for duplicates

- [ ] I have searched for similar issues before opening a new one.

### Description

in 12.3.1 keyboard nav is in the advanced playground, but it's missing from the set of files we copy over to app engine so the hosted playground is currently busted.

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```
